packages/query/src/graphql/graphql.module.ts

```diff
@@ -127,6 +128,18 @@ export class GraphqlModule implements OnModuleInit, OnModuleDestroy {
     }
   }
 
+  private setupKeepAlive(pgClient) {
+    setInterval(() => {
+      (async () => {
```

Comment:
```suggestion
      void (async () => {
```

Just to stop the linter warning